scripts/common.Makefile

```diff
@@ -48,7 +48,7 @@ VENV_DIR      := $(abspath $(REPO_BASE_DIR)/.venv)
 DOT_ENV_FILE = $(abspath $(REPO_BASE_DIR)/.env)
 
 # utils
-get_my_ip := $(shell hostname --all-ip-addresses | cut --delimiter=" " --fields=1)
+get_my_ip := $(shell (hostname --all-ip-addresses || hostname -i) 2>/dev/null | cut --delimiter=" " --fields=1)
```

Comment:
for info.. ```hostname -i``` also works on ubuntu, but returns 127.0.1.1... which is not the desired behaviour. is this returning all ips on your non-debian machine?
also a side-note, I think we only ensure compatibility with ubuntu, anything else is "at your own risk"... that is what I was told while using windows ;)